analytics/views.py

```diff
@@ -1132,6 +1133,13 @@ def support(request: HttpRequest) -> HttpResponse:
             attach_discount_to_realm(realm, new_discount)
             msg = f"Discount of {realm.string_id} changed to {new_discount} from {current_discount} "
             context["message"] = msg
+        elif request.POST.get("new_subdomain", None) is not None:
+            new_subdomain = request.POST.get("new_subdomain")
+            old_subdomain = realm.string_id
+            do_change_realm_subdomain(realm, new_subdomain)
+            msg = f"Subdomain changed from {old_subdomain} to {new_subdomain}"
+            context["message"] = msg
+            context["redirect_to"] = new_subdomain
```

Comment:
I tried this out in dev env. I think the best user experience would be to just redirect to the new_subdomain directly from that view than doing it using `<meta http-equiv>`. Can you make it work like that?